tufrepo/cli.py

```diff
@@ -152,22 +152,34 @@ def init_succinct_roles(ctx: Context, role: str):
 
 @cli.command()
 @click.pass_context
-@click.option("--target-in-repo/--no-target-in-repo", default=True)
-@click.option("--follow-delegations/--no-follow-delegations", default=True)
-@click.option("--role", default="targets")
+@click.option("--no-target-in-repo", is_flag=True, default=False)
+@click.option("--no-follow-delegations", is_flag=True, default=False)
+@click.option("--role", default="targets", metavar="ROLE", show_default=True)
```

Comment:
What does `metavar` do?
I can't find it in the `click` documentation.

Reply:
metavar is the argument content in help output "`--role ROLE`"